doc/lua_api.txt

```diff
@@ -4938,18 +4938,25 @@ This is basically a reference to a C++ `ServerActiveObject`
     * element `stat` values:
       `position`, `name`, `scale`, `text`, `number`, `item`, `dir`
 * `hud_get(id)`: gets the HUD element definition structure of the specified ID
-* `hud_set_flags(flags)`: sets specified HUD flags to `true`/`false`
-    * `flags`: (is visible) `hotbar`, `healthbar`, `crosshair`, `wielditem`,
-      `breathbar`, `minimap`, `minimap_radar`
-    * pass a table containing a `true`/`false` value of each flag to be set or
-      unset.
+* `hud_set_flags(flags)`: sets specified HUD flags of player.
+    * A table containing fields with boolean values of the HUD flags to be set or
+      unset. See `hud_get_flags` below for a list of possible flags.
     * if a flag equals `nil`, the flag is not modified
     * note that setting `minimap` modifies the client's permission to view the
       minimap - the client may locally elect to not view the minimap.
     * minimap `radar` is only usable when `minimap` is true
-* `hud_get_flags()`: returns a table containing status of hud flags
-    * returns `{hotbar=true, healthbar=true, crosshair=true, wielditem=true,
-      breathbar=true, minimap=true, minimap_radar=true}`
+* `hud_get_flags()`: returns a table of player HUD flags with boolean values.
+    * ```
+      {
+           hotbar,
+           healthbar,
```

Comment:
I don't think a code block is necessary here. Just add `Possible flags:` to the prev line and use a list:
```
* hotbar
* healthbar
```